### PR TITLE
feat: waiting message during recovery.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#326

### CODE REVIEW

File: src/common/TransactionConfirmation.test.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { render } from '../tests';
+import TransactionConfirmation, { Status, Props } from "./TransactionConfirmation";
+
+fdescribe("TransactionConfirmation", () => {
```
Comment (gdethier) [inline src/common/TransactionConfirmation.test.tsx:4]: remove the `f`.

File: src/common/TransactionConfirmation.test.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        const result = render(<TransactionConfirmation { ...props } />);
+        const expectedResult = `
+<p>
+  ${ expectedStatus }
+</p>
+`;
+        expect(result).toMatchInlineSnapshot(expectedResult);
+    }
+
+    test("goes to status TRANSFERRING upon start", () => {
```
Comment (gdethier) [inline src/common/TransactionConfirmation.test.tsx:16]: Don't we use `it` in the other tests?

**Review by gdethier — CHANGES_REQUESTED** [review]
Great job! Just a couple of suggestions and one change.

File: src/common/TransactionConfirmation.test.tsx (hunk 1 above)
Comment (benoitdevos) [reply src/common/TransactionConfirmation.test.tsx]: Done.

File: src/common/TransactionConfirmation.test.tsx (hunk 2 above)
Comment (benoitdevos) [reply src/common/TransactionConfirmation.test.tsx]: Yes, of course. Done.

**Review by gdethier — APPROVED** [review]
:+1:

**Pull request merged**